File: 7_Kyu_Kata/shorter-concat-reverse-longer/Python/shorter_concat_reverse_longer.py
# [object Object] - shorter concat [reverse longer]  [ ID: 54557d61126a00423b000a45  (shorter-concat-reverse-longer) ]
# URL: https://www.codewars.com/kata/54557d61126a00423b000a45
# Category: REFERENCE  |  Tags: STRINGS | FUNDAMENTALS
# ******************************************************************************
shorter_reverse_longer = lambda a, b: b + a[::-1] + b if len(a) >= len(b) else a + b[::-1] + a

File: kata-7-kyu/shorter-concat-reverse-longer/javascript/shorterConcatReverseLonger.js
//+ ====================================================================================================================
//+
//+ 7 kyu - shorter concat [reverse longer]  [ ID: 54557d61126a00423b000a45 ] (shorter-concat-reverse-longer)
//+ URL: https://www.codewars.com/kata/54557d61126a00423b000a45
//+ Category: REFERENCE  |  Tags: STRINGS | FUNDAMENTALS
//+
//+ ====================================================================================================================

const shorter_reverse_longer = (a, b) =>
  a.length >= b.length ? b + [...a].reverse().join("") + b : a + [...b].reverse().join("") + a

module.exports = { shorter_reverse_longer }
